### PR TITLE
Feature#37 views for appointments

### PR DESCRIPTION
**Pull request by martinmt-mx**
# Description
 - The styles for the Appointments views are responsive.
 - Now there is a better visualization of the status of each of the Appointments.
 - The Edit view has also been created to edit the Appointments.. 
 - Several errors that had conflicts with this new feature have been corrected.
  
  ## Screenshots 
image
image

### CODE REVIEW

**Review by oramasangel — DISMISSED** [review]
lgtm

**Review by JoseRicardoJimenez — DISMISSED** [review]
Good job!

File: app/views/appointments/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      <div class="actions mt-3">
+        <%= f.submit "Save", class: "btn btn-outline-primary d-block w-25 mx-auto mb-2" %>
+      </div>
+
+      <div class="btn_cancel">
+        <%= link_to t('cancel'), appointments_path, class:"btn toolbar_back"%>
+      </div>
+    <% end %>
+  </div>
+</div>
```
Comment (YilKanda) [inline app/views/appointments/edit.html.erb:39]: Add a blank line

File: app/views/appointments/index.html.erb [hunk 2]
```diff
@@ -1 +1,6 @@
-<%= render 'list' %>
\ No newline at end of file
+<div class="table-container">
+    <div class="title">
+        <h2> <%= t('my_appointments') %> </h2>
+    </div>
+    <%= render 'list' %>
+</div>
```
Comment (YilKanda) [inline app/views/appointments/index.html.erb:6]: Add a blank line

**Review by YilKanda — COMMENTED** [review]
Check the indentation of your style files, please.

**Review by xFerchoVJ — APPROVED** [review]

**Review by SaraNavarro — APPROVED** [review]

**Review by SampabloC — APPROVED** [review]

**Pull request merged**